lib/board.rb

```diff
@@ -0,0 +1,80 @@
+require "pry"
+
+class Board
+
+  attr_accessor :cells, :player, :token_1, :token_2
+
+  def initialize
+    @cells = Array.new(9, " ")
```

Comment:
You already have this code in the `reset!` method. No need to duplicate it; just call `reset!` from here.